newdle/client/src/components/creation/FinalStep.js

```diff
@@ -13,6 +13,8 @@ import client from '../../client';
 import {newdleCreated, setStep, setTitle} from '../../actions';
 import styles from './creation.module.scss';
 
+const MAX_INPUT_LENGTH = '80';
```

Comment:
does using a number cause any warnings?

Reply:
I thought it had to be a string but guess not